content/blog/how-to-use-npm/index.md

```diff
@@ -0,0 +1,574 @@
+
+> You can [learn more about what a runtime is and how they work from our article that introduces the concept](/posts/how-computers-speak/#compiled-vs-runtime)
+
+Node also comes with an advantage over browsers for running JavaScript: you can interface with lower-level programming languages such as C via [Node's N-API](). This means that libraries you rely on can build on top of this N-API to provide a way to do things like send native desktop notifications, show something particular in your taskbar, or any other action that would require lower-level access to your local machine than JavaScript typically provides.
+
+### `npm` {#whats-npm}
+
+Any sufficiently useful programming language needs an ecosystem to rely on. One of the primary elements for an ecosystem is a collection of libraries that you can use to build out your own libraries and applications.
+
+> A library is a snippet of code that other people have written that you can easily import into your own code and use yourself - often with a single line of code
+
+`npm` is a combination of two things:
+
+1) The registry - the servers and databases that host the packages with their specific named packages. 
+2) The client-side CLI utility - the program that runs on your computer in order to install and manage the packages on your local disk
+
+When, say, Facebook wants to publish a new version of `react`, someone from the React team (with publishing credentials) will setup and build the production version of the React source code, open the client-side utility in order to run the command `npm publish`, which will send the production code to the registry. From there, when you install `react` using the `npm` command on your device, it will pull the relevant files from the registry onto your local machine for you to use.
+
+While the registry is vital for the usage of the CLI utility, most of the time we say `npm` in this article, we're referring to the CLI tool. We'll make sure to be explicit when talking about the registry itself
+
+## Setting Up Node {#setup-node}
+
+Before we explain how to install Node, let's explain something about the release process of the software. 
+
+When it comes to install options there are two: 
+
+1) LTS
+
+2) Current
+
+The "LTS" release stands for "long-term release" is considered the most "stable" release, and is recommended for production usage. This is because LTS releases will receive critical bug fixes and improvements even after a new version comes along. LTS releases often see years of support.
```

Comment:
LTS is long term support

Reply:
```suggestion
The "LTS" release stands for "long-term support" and is considered the most "stable" release that is recommended for production usage. This is because LTS releases will receive critical bug fixes and improvements even after a new version comes along. LTS releases often see years of support.
```